backend/src/main/java/se/chalmers/dat265/group1/model/DataModel.java

```diff
@@ -0,0 +1,8 @@
+package se.chalmers.dat265.group1.model;
+/**
+ * Created by ciff on 13/10/2017.
+ */
+public abstract class DataModel<T> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') T is not used in the class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "DataModel" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)